inductor: Test for crash when nn.module has a bad getattr call.

c00w opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #145799

Note that I can't get this working, so I'd love some advice on how to write
this test.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames

c00w committed:
inductor: Test for crash when nn.module has a bad getattr call.

Note that I can't get this working, so I'd love some advice on how to write
this test.

[ghstack-poisoned]

c00w committed:
Update on "inductor: Test for crash when nn.module has a bad getattr call."


Note that I can't get this working, so I'd love some advice on how to write
this test.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

jansel commented on test/dynamo/test_compile.py:
```diff
@@ -188,6 +188,30 @@ def fn(x):
+
+        x = torch.randn(10, 10)
+        # with self.assertRaises(AttributeError):
+        fn(testModel(), x)
```
Looks like the test is working in CI... what is the issue you are seeing?

c00w replied:
It's not hitting this crash path (i.e. not throwing a InternalDynamoException error), and logging confirms it's not hitting the crash path.

c00w committed:
Update on "inductor: Test for crash when nn.module has a bad getattr call."


Note that I can't get this working, so I'd love some advice on how to write
this test.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

c00w commented:
Answer was turning off inline_inbuilt_nn_modules
